modules/benchmarks/pom.xml

```diff
@@ -35,6 +35,7 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
+        <jmh.version>1.19</jmh.version>
```

Comment:
You shouldn't change jmh version here. The right place is `ignite/parent/pom.xml`.